server/chat-commands/core.ts

```diff
@@ -1275,13 +1309,15 @@ exports.commands = {
 			return;
 		}
 		if (!target) return this.errorReply("Provide a valid format.");
-		let originalFormat = Dex.getFormat(target);
-		// Note: The default here of [Gen 7] Pokebank Anything Goes isn't normally hit; since the web client will send a default format
-		let format = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat('[Gen 7] Pokebank Anything Goes');
+		const originalFormat = Dex.getFormat(target);
+		// Note: The default here of [Gen 8] Anything Goes isn't normally hit; since the web client will send a default format
+		const format = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat(
+			'[Gen 7] Pokebank Anything Goes'
```

Comment:
You changed the comment to [Gen 8] Anything Goes but the actual code still refers to [Gen 7] Pokebank Anything Goes... it's a good thing that this code path is never taken!